packages/components/package.json

```diff
@@ -46,6 +46,7 @@
     "react-markdown": "^4.1.0",
     "react-popper": "^2.2.5",
     "react-router-dom": "^5.2.0",
+    "rrule": "^2.6.8",
```

Comment:
From our chat. Don't need this anymore. uninstall it and once that's done, run `npm ci` again on the root folder since lerna is a silly goose that does something with the package locks

Reply:
exactly. this was needed for `computeOccurrence` which has been cut from the scope of this PR

can be removed now